src/app/pages/CreateConnectorPage/StepReview.tsx

```diff
@@ -284,14 +281,16 @@ export const StepReviewComponent: React.FC<StepReviewComponentProps> = ({
                       <strong>{_.startCase(el)}</strong>
                     </GridItem>
                     <GridItem span={8}>
-                      {dataToHide.includes(el) ? (
-                        maskValue(modifiedObject[el])
-                      ) : typeof modifiedObject[el] === 'object' ? (
+                      {typeof modifiedObject[el] === 'object' ? (
```

Comment:
I wonder if pulling this into a function would make it easier to understand.  Then you could maybe do a `switch (typeof modifiedObject[el]` which would simplify at least one ternary operator.

Reply:
sure